lambda/MAS/Views/ContentView.cshtml

```diff
@@ -0,0 +1,87 @@
+﻿@model MAS.Models.Item
+
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <meta charset="utf-8" />
+    <meta name="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>
+        @Model.Title
+    </title>
+
+    <meta name="description" content="@Model.ShortSummary." />
+
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@("@nhs_sps")" />
+    <meta name="twitter:title" content="@Model.Title" />
+
+    <meta property="og:locale" content="en_GB" />
+    <meta property="og:site_name" content="Medicines awareness service" />
+    <meta property="og:type" content="article" />
+    <meta property="og:url" content="https://www.medicinesresources.nhs.uk/en/Medicines-Awareness/Press-and-Media/Press-and-media/@Model.Slug/" />
+    <meta property="og:title" content="@Model.Title" />
+    <meta property="og:description" content="@Model.ShortSummary" />
+    <meta property="og:image" content="https://www.medicinesresources.nhs.uk/open-graph.png" />
+    <meta property="og:image:type" content="image/png" />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="og:image:alt" content="Specialist Pharmacy Service logo" />
+
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700"
+          rel="stylesheet"
+          type="text/css" />
+    <link href="/app.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+    <header>
+        <a href="https://www.sps.nhs.uk/" target="_blank" rel="noopener" aria-label="Sepcialist Pharmacy Service website">
+            <img src="/sps-logo.svg"
+                 alt="Sepcialist Pharmacy Service logo"
+                 class="logo" />
+        </a>
+    </header>
+    <main>
+        <article aria-labelledby="title" aria-describedby="description">
+            <h1 id="title">
+                <a href="@Model.URL"
+                   target="_blank" rel="noopener">
+                    @Model.Title
+                </a>
+            </h1>
+
+            <p id="description">
+                @Model.ShortSummary
+            </p>
+
+            <div class="comment">
+                <h2>SPS commentary:</h2>
+                <p>
+                    @Html.Raw(@Model.Comment)
```

Comment:
I think a comment can be HTML, in which case it will already have paragrahs etc inside it, in which case you don't need to the extra `p` tag wrapping it

Reply:
Also, comments are optional so this needs to only be rendered if it exists